crates/red_knot_project/src/metadata/pyproject.rs

```diff
@@ -121,7 +119,7 @@ pub enum ResolveRequiresPythonError {
     TooLargeMajor(u64),
     #[error("The minor version `{0}` is larger than the maximum supported value 255")]
     TooLargeMinor(u64),
-    #[error("value `{0}` does not contain a lower bound. Add a lower bound to indicate the minimum compatible Python version (e.g., `>=3.13`).")]
+    #[error("value `{0}` does not contain a lower bound. Add a lower bound to indicate the minimum compatible Python version (e.g., `>=3.13`) or specify a version in `environment.python-version`.")]
```

Comment:
They might not have a `tool.knot` table at all, right? This could be quite confusing if they aren't familiar with how to provide this red-knot-specific setting

```suggestion
    #[error("value `{0}` does not contain a lower bound. Add a lower bound to indicate the minimum compatible Python version (e.g., `>=3.13`) or specify a version in `tool.knot.environment.python-version`.")]
```